lib/applicants/resolvers.ts

```diff
@@ -17,18 +18,18 @@ export const applicants: Resolver = async (_parent, _args, { prisma }) => {
  * @returns Status of operation (ok, error)
  */
 export const createApplicant: Resolver = async (_, args, { prisma }) => {
-  const {
-    input: { email },
-  } = args;
+  const { input } = args;
 
   let applicant;
   try {
+    input.postalCode = formatPostalCode(input.postalCode);
+    input.phone = formatPhoneNumber(input.phone);
     applicant = await prisma.applicant.create({
       data: { ...args.input },
     });
```

Comment:
We don't want to modify the input args to this function as it could cause debugging issues down the road and is generally bad practice. Instead, we could do something like this to format the fields without modifying the function parameters:
```suggestion
    applicant = await prisma.applicant.create({
      data: { 
          ...args.input,
          postalCode: formatPostalCode(input.postalCode),
          phone: formatPhoneNumber(input.phone),
       },
    });
```

Reply:
good point, I like this a lot better